### PR TITLE
Update readme and remove unused code

### PR DESCRIPTION
**Pull request by IldarKamalov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,11 @@ parameter | explanation
 `stealth.user_agent` |  A string representing a user agent that is used by the stealth module as a user agent value. It can be an empty string. If this query parameter does not exist, it is treated as not enabled.
 `stealth.ip` |  A string representing a IP address that is used by the stealth module as a ip address. If this query parameter does not exist, it is treated as not enabled.
 `stealth.dpi` | Can be `true` or `false`.
+`stealth.block_trackers` | Can be `true` or `false`.
```
Comment (slavaleleka) [inline README.md:47]: this stealth option enables AdGuard Tracking Protection filter so there is no need to pass it to the ReportsWebApp

File: README.md [hunk 2]
```diff
@@ -44,6 +44,11 @@ parameter | explanation
...
+`stealth.disable_wap_push_message_routing_service` | Can be `true` of `false`.
```
Comment (slavaleleka) [inline README.md:48]: what stealth options is this parameter for?

Comment (IldarKamalov) [reply README.md:48]: Screenshot 2023-10-20 at 10 16 22

This is the new Windows tracking settings

File: README.md (hunk 1 above)
Comment (IldarKamalov) [reply README.md:47]: Screenshot 2023-10-20 at 10 18 42

According to the new design, we display this setting in step 4

Comment (slavaleleka) [reply README.md:47]: maybe better to check if filter id `3` is passed? 
the same for "Remove tracking parameters from URLs" (related filter id is 17)

File: README.md (hunk 2 above)
Comment (Chinaski1) [reply README.md:48]: @slavaleleka [For example this feature on macOS.](https://uploads.adguard.org/Chinaski71n2c.png)

**Pull request merged**